pkg/config/config_suite_test.go

```diff
@@ -1,4 +1,4 @@
-package config_test
+package config
```

Comment:
This shouldn't have been changed I guess

Reply:
Right, I think GoLand is messing those up constantly.

Reply:
Fixed ae5fb65